.github/workflows/ci.yml

```
@@ -22,10 +22,21 @@ jobs:
       uses: actions/checkout@v2
       with:
         submodules: true
-    - name: Build ${{ matrix.distro }} Docker image with ${{ matrix.compiler }}
+    - name: Prepare base image in Docker
+      run: docker build . -f utils/docker/Dockerfile.${{ matrix.distro }}
+                          --build-arg DAOS_DEPS_BUILD=no
+    - name: Build dependencies in Docker
+      run: docker build . -f utils/docker/Dockerfile.${{ matrix.distro }}
+                          --build-arg DEPS_JOBS=10 --build-arg DAOS_BUILD=no
+    - name: Build Docker image with ${{ matrix.compiler }}
       run: docker build . -f utils/docker/Dockerfile.${{ matrix.distro }}
                           --build-arg COMPILER=${{ matrix.compiler }}
-                          -t ${{ matrix.distro }}/${{ matrix.compiler }}/daos:${{ github.sha }}
+                          --build-arg DEPS_JOBS=10 --build-arg DAOS_JAVA_BUILD=no
+    - name: Build Java in Docker image with ${{ matrix.compiler }}
```

```suggestion
    - name: Build DAOS Java client in Docker image with ${{ matrix.compiler }}
```
Or maybe somebody can suggest something even more accurate instead of _DAOS Java client_.